service/history/worker_versioning_util.go

```diff
@@ -219,6 +219,15 @@ func updateWorkflowAssignedBuildId(
 	)
 }
 
+func MakeDirectiveForWorkflowTask(ms workflow.MutableState) *taskqueuespb.TaskVersionDirective {
```

Comment:
just wondering: should this and MakeDirectiveForActivityTask be methods on MutableState?

Reply:
it's a util function wrapping another one, I'm not certain that it's a good candidate to be added to the MutableState interface. (at least yet)

Reply:
yeah, that's fair